clippy_lints/src/nonstandard_macro_braces.rs

```diff
@@ -16,8 +16,7 @@ declare_clippy_lint! {
     /// Checks that common macros are used with consistent bracing.
     ///
     /// ### Why is this bad?
-    /// This is mostly a consistency lint although using () or []
-    /// doesn't give you a semicolon in item position, which can be unexpected.
+    /// This is mostly a consistency lint.
```

Comment:
This section isn’t really helpful. “Having non-conventional braces on well-stablished macros can be confusing when debugging, and they bring incosistencies with the rest of the ecosystem.” Or something like it would be better, what do you think about it?


```suggestion
    /// Having non-conventional braces on well-stablished macros can be confusing when debugging,
    /// and they bring incosistencies with the rest of the ecosystem.”

```